dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEventAnalyticsManager.java

```diff
@@ -580,6 +570,44 @@ else if ( params.isOrganisationUnitMode( OrganisationUnitSelectionMode.CHILDREN
         return sql;
     }
 
+    /**
+     * Creates a SQL statement of descendants org units. When there are multiple
+     * levels the "or" operator will be used as junction.
+     *
+     * @param orgUnitAlias
+     * @param dimensionalItems
+     * @param helper
+     * @return a SQL statement in the format "where/and (ax."uidlevel0" in
+     *         ('orgUid-1', 'orgUid-2', 'orgUid-2') )"
+     */
+    private String descendantsOrgUnitStatement( final String orgUnitAlias,
+        final List<DimensionalItemObject> dimensionalItems, final SqlHelper helper )
+    {
+        final StringBuilder statement = new StringBuilder();
+        final Map<String, String> orgUnitColsAndUnitUids = new HashMap<>();
+
+        statement.append( helper.whereAnd() ).append( " (" );
+
+        for ( final DimensionalItemObject itemObject : dimensionalItems )
+        {
+            final OrganisationUnit unit = (OrganisationUnit) itemObject;
+            final String orgUnitCol = quote( orgUnitAlias, "uidlevel" + unit.getLevel() );
+            final String orgUnitValue = isBlank( orgUnitColsAndUnitUids.get( orgUnitCol ) )
+                ? wrap( unit.getUid(), "'" )
+                : trimToEmpty( orgUnitColsAndUnitUids.get( orgUnitCol ) ) + ", " + wrap( unit.getUid(), "'" );
+
+            orgUnitColsAndUnitUids.put( orgUnitCol, orgUnitValue );
+        }
+
+        // Generates a code like: ax."uidlevel0" in ('orgUid-1', 'orgUid-2',
```

Comment:
Remove this comment and rely on the main javadoc for the method instead.

Reply:
Done